### DISCUSSION TITLE
proposal: can I add a caution about 'git config --unset hooksPath' on installation wiki?

### DISCUSSION THREAD

**Discussion by ghost** [evilmartians/lefthook#379] [Ideas]
hi thank you for this great product. Up to now I really like this, but one thing to scare me is that lefthook copies the existing git template to  "*.old" and create a new git template.

Just think, if you have a script to check if there is a private things in pre-commit, you'll skip that check when you run lefthook. I wanted to know if this causes such a behavior before I try, someone will be in the same case and I worry if some of them might feel scared and stop using lefthook.

beside adding a caution, how about adding `source {hooksPath}/lefthook.pre-commit` into existing git_template? In this case, you can check the file is applied to lefthook easily and never mess the original one. I know I need to care about the case if original files are not written in posix, though.

**Comment by mrexox** [comment]
Hi! Sorry for the long time waiting. Your concern makes sense, but I think that the purpose of lefthook is to be the tool to control all the hooks, and custom backup'ed hooks are not meant to be controlled. It is better to rewrite them with `lefthook-local.yml` and keep things in a system.
I suppose it would be great to convert these hooks into `scripts` and automatically generate `lefthook.yml` for them. WDYT? I'll convert this issue into a discussion.
